src/content/docs/full-stack-observability/observe-everything/get-started/get-started-auto-telemetry-pixie.mdx

```diff
@@ -0,0 +1,95 @@
+---
+title: Get started with Auto-telemetry with Pixie
+tags:
+  - Pixie Auto-telemetry
+  - Service monitoring
+  - Kubernetes
+  - eBPF
+metaDescription: 
+watermark: BETA
+redirects:
+  - /docs/full-stack-observability/get-started-auto-telemetry-pixie
+---
+
+<Callout variant="important">
+Auto-telemetry with Pixie is a supported public beta. Therefore, the features and functionality are subject to change. By joining the beta for Auto-telemetry with Pixie, you agree to the terms in the [New Relic Pre-Release Policy](/docs/licenses/license-information/referenced-policies/new-relic-pre-release-policy/). 
+</Callout>
+ 
+When we say auto-telemetry, we’re not talking about cars &mdash; we’re talking about automatically collecting telemetry data for your Kubernetes clusters. With our [Pixie](https://pixielabs.ai/) integration, you no longer have to manually instrument your clusters or change code. With one install command, you get observability into your Kubernetes cluseters and workloads. No language agents required. 
+
+Pixie collects telemetry data using [eBPF](https://ebpf.io/), a virtual machine-like construct in the Linux kernel. eBPF makes it easy to collect similar data about your clusters and workloads as with traditional New Relic APM agents, but now without needing to update your code.
```

Comment:
eBPF is strange, because it feels like an acronym, but I don't think there's any there there, acronym wise. I did find "extended Berkeley Packet Filter" but maybe it's not worth exploding here?

Reply:
You're calling them APM agents here and you called them language agents above. Maybe make these consistent?

Reply:
I sort of want to shove some of this technical detail farther down in this doc, maybe in a "How it works" section?